src/views/login.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div class="login__container">
+    <form @submit.prevent="loginUser">
+      <div class="container">
+        <label for="email"><b>Email</b></label>
+        <input
+          v-model="email"
+          type="text"
+          placeholder="Enter Email"
+          name="email"
+          id="email"
+          required
+        />
+        <label for="psw"><b>Password</b></label>
+        <input
+          v-model="password"
+          type="password"
+          placeholder="Enter Password"
+          name="psw"
+          required
+        />
+        <button type="submit">Login</button>
+        <label>
+          <input type="checkbox" checked="checked" name="remember" /> Remember
+          me
+        </label>
+      </div>
+      <div class="container" style="background-color: #f1f1f1">
+        <button type="button" class="cancelbtn">Cancel</button>
+        <router-link to="/register"
+          ><span class="psw"
+            >Not Registred?<a href="#">Register</a></span
+          ></router-link
+        >
+      </div>
+    </form>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    loginUser() {
+      axios
+        .post(
+          "https://documentation-vue.projects.codennection.pl/api/auth/login",
```

Comment:
Główny adres api czyli https://documentation-vue.projects.codennection.pl powinien być ustawiony w konfiguracji axiosa, tak abyśmy nie musieli go powtarzać przy każdym requeście. Po skonfigurowaniu takiego adresu, będziemy wysyłać request na adres /api/auth/login. Instrukcja:
https://stackoverflow.com/questions/48001813/best-way-to-config-global-headers-for-get-post-patch-in-vuejs

Reply:
let myApi = axios.create({
  baseURL: 'https://documentation-vue.projects.codennection.pl/api/auth/login',
  timeout: 3000,
  **headers: {'X-Custom-Header': 'CustomHeader1'}** <<<< CO TO JEST?
}); + dobrym zabiegiem jest wpisanie to w computed ?

Reply:
Stwórz konfigurację axiosa w osobnym pliku. Stwórz go w  @/axios/axios.js i tam skonfiguruj axiosa i wyexportuj. Następnie wszedzie gdzie używasz axiosa musisz zaimportować go z tej lokalizacji czyli:
import axios from '@/axios/axios.js' 

Przykład pliku @/axios/axios.js:
https://www.digitalocean.com/community/tutorials/vuejs-rest-api-axios#creating-a-common-base-instance